### ISSUE TITLE
Train on my own dataset

### ISSUE THREAD

**Issue by disturbed-mystic1** [LS4GAN/uvcgan#14]
Is there any implementation of this using an .ipynb notebook. An implementation of this would be very helpful.

**Comment by usert5432** [comment]
Hi @disturbed-mystic1,

Unfortunately, we have not developed any notebooks for this project.

However, if you wish to train `uvcgan` on your own dataset, you could try taking one of our scripts as a starting point, e.g. [this one should work](https://github.com/LS4GAN/uvcgan/blob/main/scripts/train/celeba/cyclegan_celeba_preproc-256.py).  And modify it to your needs. We will be happy to assist you if you run into any problems.

**Comment by Deepika1498** [comment]
What does align_train = false mean in the training script?

**Comment by usert5432** [comment]
Hi @Deepika1498,

> What does align_train = false mean in the training script?

This option makes sense only when you have an aligned dataset. An aligned dataset, is the dataset where all images are paired. That is, for each source image you know how its translation should look like. An example of the aligned dataset is a dataset of satellite images (domain A) and the corresponding maps (domain B), as illustrated [here](https://phillipi.github.io/pix2pix/images/teaser_v3.png).

If you have an aligned dataset, setting `align_train = True` will load the images in pairs. So, the network can see exactly how the source and translation should look like. If you set `align_train = False`, then the images will be shuffled, and so the network will not know how the translation of the image should look like.

If the dataset is not aligned (like CelebA), this option is largely irrelevant and can be set to any value.

**Comment by Deepika1498** [comment]
Can uvcgan be used to perform image fusion of satellite images to improve the spatial resolution of the landsat images by training on 30m landsat and 10m sentinel data?

**Comment by usert5432** [comment]
Hi @Deepika1498, I am not familiar with the problem of image fusion of satellite images, so I am not able to answer this question.

I may be able to provide more information on the feasibility of the UVCGAN usage for the problem of image fusion, if you could elaborate on your training setup?

**Comment by MajidHeydari** [comment]
I want to set up with satellite dataset with this:http://weegee.vision.ucmerced.edu/datasets/landuse.html

**Comment by bye111** [comment]
请问我训练遇到OSError: Failed to find fully trained model in 'outdir\celeba_preproc/model_d(cyclegan)_m(cyclegan)_d(basic)_g(vit-unet)_cyclegan_vit-unet-12_male2female_self-lsgan-paper-cycle_mid-256' for transfer: [Errno 2] No such file or directory: 'outdir\\celeba_preproc/model_d(cyclegan)_m(cyclegan)_d(basic)_g(vit-unet)_cyclegan_vit-unet-12_male2female_self-lsgan-paper-cycle_mid-256\\net_gen_ab.pth'
是什么原因，似乎无法生成pth文件
